assets/stylesheets/components/_team.scss

```diff
@@ -1,42 +1,44 @@
-.team {
-
-	.member {
-		@include box;
-		float: left;
-		width: (100% / 3);
-		padding: 10px;
-		text-align: center;
-		border-left: solid 1px #ccc;
-		border-top: solid 1px #ccc;
```

Comment:
Flexbox is already noted as an issue #58 for you :P 

As for colors, waiting for the finals from the styleguide and I don't think this has the new color management system.

As a whole, I expect a ton of this has to be refactored. It is really just quick and dirty structural layout aimed to give a high level overview of what we envisioned.